File: php/examples/get_resource.php
<?php
require_once 'base_request.php';

// Example of GET /resources/:resource_id

$debug = true;
$resource_id = $_ENV['AGENCY_RESOURCE_ID'];
$request_data = '';
$request_params = [];
$url_path = '/resources/' . $resource_id;

send_request('GET', $url_path, $request_params, $request_data);

File: csharp/examples/GetResource.cs
using System;
namespace Examples.examples
{
    // Example of GET /resources/:resource_id
    
    public class GetResource : BaseRequest
    {
        private string _resourceId;

        public GetResource()
        {
            _resourceId = Environment.GetEnvironmentVariable("AGENCY_RESOURCE_ID");
            Console.WriteLine($"GetResource: _resourceId={_resourceId}");
        }

        public override void SendRequest()
        {
            GetRequest($"resources/{_resourceId}");
        }
    }
}

